### ISSUE TITLE
Support for multiple languages

### ISSUE THREAD

**Issue by scribblemaniac** [dkhamsing/open-source-ios-apps#392]
It may be nice to have a way to include other languages in our repository. Of course people can just replace all of the English with another language if they want to translate, but it might be nice to set up a better standard for this so that multiple languages can be added, especially now that we are looking at more dynamic options (mobile app, webpage) that could allow the user to switch languages. Additionally, there some fields that are the same for all languages so duplicating the file for every language is a bit of a waste of space.

I would love to hear other people's thoughts on how this should be implemented. So far I am thinking that we should handle translations with a standard translation format (XLIFF, TS, or something like that), and then integrate an existing parsing library into our programs. The advantage of this is that it follows an existing standard and will be easier for people to translate since there are already graphical tools to help people translate with those formats. The disadvantage is that it is a completely different format than the main content. Alternatively we could add ids to all the items (something simple like lowercase title with dashes for spaces so we can completely automate that) and then there could be <language>.json files that follow the same structure as the main file, but only specify the id, and any fields that need to be overridden for translation. The advantage of this is the familiar format, and probably compactness. Examples:

Original
```
{{
  "title": "Moves",
  "id": "moves",
  "category": "location",
  "description": "Visualize where you spend your time",
  "license": "mit",
  "source": "https://github.com/neonichu/Places",
  "stars": 33,
  "date_added": "Wed Jul 13 06:47:19 2016 -0700",
  "suggested_by": "@dkhamsing"
}
```

XLIFF (sorta)
```
<file original="content.json"  source-language="en-US" target-language="fr-CA" datatype="json">
 <trans-unit>
  <source>Visualize where you spend your time</source>
  <target>Visualizez où vous passez votre temps</target>
 </trans-unit>
</file>
```

JSON file (fr-CA.json)
```
{
  "id": "moves",
  "description": "Visualizez où vous passez votre temps"
}
```

**Comment by dkhamsing** [comment]
Would be an interesting project

I would prefer if we can keep contents.json as the base of the English README

Is there a way of doing this by only adding "id" for the entries that are translated

**Comment by scribblemaniac** [comment]
> Is there a way of doing this by only adding "id" for the entries that are translated

Sure. In fact, the id can be implied for most cases. For example we could specify the id as a lowercase version of the title, with a dash instead of spaces. Then the only circumstances where you would actually need to specify is when the title is changed, or maybe if it contains special characters. The rest of the time, the programs can generate the id when needed from the title.

**Comment by dkhamsing** [comment]
Maybe we can secure at least one language translation effort before we start.. 

@wg689 we are interesting in building support for multiple languages right from this project
could you help with the Chinese translation?

**Comment by dkhamsing** [comment]
@scribblemaniac will be closing this unless we can get someone to translate

**Comment by dkhamsing** [comment]
Closing for now
